src/sagemaker/utils.py

```diff
@@ -608,3 +608,22 @@ def __getattr__(self, name):
             name:
         """
         raise self.exc
+
+
+def _module_import_error(py_module, feature, extras):
+    """Return error message for module import errors, provide
+    installation details.
+
+    Args:
+        py_module (str): Module that failed to be imported
+        feature (str): Affected sagemaker feature
+        extras (str): Name of the extra_requirements to install all of the dependencies
```

Comment:
* s/extra_requirements/`extras_require` (with backticks)
* s/all of the/the relevant